src/main/java/fr/openent/competences/helper/DateHelper.java

```diff
@@ -0,0 +1,44 @@
+package fr.openent.competences.helper;
+
+import fr.openent.competences.constants.DateFormat;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class DateHelper {
+    private static final Logger log = LoggerFactory.getLogger(DateHelper.class);
+
+    private DateHelper() {
+        throw new IllegalStateException("Utility class");
```

Comment:
unreachable statement and just not necessary.

Reply:
Yes it is: https://rules.sonarsource.com/java/RSPEC-1118/